javaScript/index.js

```diff
@@ -41,4 +41,120 @@ function handleScreenChange(event) {
 
 handleScreenChange(mobileQuery);
 
-mobileQuery.addEventListener('change', handleScreenChange);
\ No newline at end of file
+mobileQuery.addEventListener('change', handleScreenChange);
+
+const projects = [
+  {
+    cardName: 'Multi-Post Stories Gain+Glory',
+    technologies: `
+    <li>Ruby on Rails</li>
+    <li>CSS</li>
+    <li>JavaScript</li>
+    <li>HTML</li>`,
+    cardSource: '<p class="btn"><a href="#" id="popupbtn1">See project</a></p>',
+  },
+  {
+    cardName: 'Multi-Post Stories Gain+Glory',
+    technologies: `
+    <li>Ruby on Rails</li>
+    <li>CSS</li>
```

Comment:
- [ ] Kindly make sure your array of objects contains the correct (essential) elements it is supposed to for best practices. you are not allowed to store the `HTML` elements in your objects. Kindly fix that.

hint:
```js
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```